### ISSUE TITLE
I don't think you have solved the last question in Chapter 15 very well.

### ISSUE THREAD

**Issue by ShenTensen** [pezy/CppPrimer#140]
### Exercise information

Exercise 15.39

### Your enviroment information

- **System**:  :grey_question:
- **Compiler version/IDE**:  :grey_question:

**Comment by zhqu1148980644** [comment]
Yeah, Still need a complete implementation of `query` function in `TextQuery` class to ensure the translation of text typed by user  to real operation with AND OR NOT class.
